docs/proposals/tracing.md

```diff
@@ -111,28 +115,47 @@ Jaeger is the most popular tracer of OpenTracing. List of supported tracers in O
 type: Mesh
 name: default
 tracing:
-  sampling: 10.0 # percentages 0-100
-  backend:
-    openCensus:
-      stackdriver:
-        projectId: 1234
-        address: # optional
-      zipkin:
-        url: http://127.0.0.1:9411/api/v2/spans
-      agent:
-        address: "ipv4:127.0.0.1:345" # https://github.com/grpc/grpc/blob/master/doc/naming.md
-      incomingTraceContext: # in what header format trace will be consumed. Default all, Envoy looks for all of them.
-        - b3
-        - w3c
-        - cloud
-      outgoingTraceContext: # in what header format trace will be produced (default: b3)
-        - b3
+  backends:
+    - name: my-oc
+      sampling: 10.0 # percentages 0-100
+      openCensus:
+        stackdriver:
+          projectId: 1234
+          address: # optional
+        zipkin:
+          url: http://127.0.0.1:9411/api/v2/spans
+        agent:
+          address: "ipv4:127.0.0.1:345" # https://github.com/grpc/grpc/blob/master/doc/naming.md
+        incomingTraceContext: # in what header format trace will be consumed. Default all, Envoy looks for all of them.
+          - b3
+          - w3c
+          - cloud
+        outgoingTraceContext: # in what header format trace will be produced (default: b3)
+          - b3
 ```
 
 It seems that you can pick multiple Open Census exporters at once.
 Since exporters here are limited to Stackdriver and Zipkin you can send traces to Agent which will send it to all supported backends listed [here](https://opencensus.io/exporters/supported-exporters/go/ocagent/).
 
+### TrafficTrace
+
+Additionally, to enable Tracing, you have to select Dataplanes that tracing will be enabled on.  
+
+```yaml
+type: TrafficTrace
+mesh: default
+name: us
+selectors:
```

Comment:
`selectors` of what ?

Reply:
of the dataplanes, just like in ProxyTemplate